bc_obps/common/permissions.py

```diff
@@ -103,7 +99,11 @@ def get_permission_configs(permission: str) -> Optional[Union[Dict[str, List[str
             'authorized_app_roles': ["industry_user"],
             'authorized_user_operator_roles': ["admin"],
         },
-        "authorized_irc_user": {
+        "v1_authorized_irc_user_and_industry_admin_user": {
+            'authorized_app_roles': all_authorized_app_roles,
+            'authorized_user_operator_roles': ["admin"],
+        },
+        "v1_authorized_irc_user": {
```

Comment:
I’m curious about the reason for adding `v1_` to these permission groups. 🤔

Reply:
I was thinking that reg2 would have totally different cas user groups. Upon reflection, everyon can view everything, so the `GET`s could still use these. I'll ditch the prefix